lib/ansible/modules/packaging/os/yum.py

```diff
@@ -1058,6 +1084,10 @@ def ensure(module, state, pkgs, conf_file, enablerepo, disablerepo,
 
     dis_repos =[]
     en_repos = []
+
+    if skip_broken:
+        yum_basecmd.extend(['--skip-broken'])
```

Comment:
Yep https://linux.die.net/man/8/yum --skip-broken is in the general section meaning it can be added to any other action.